components/eam/src/physics/crm/physpkg.F90

```diff
@@ -1,3033 +1,1829 @@
 module physpkg
-  !-----------------------------------------------------------------------
-  ! Purpose:
-  !
-  ! Provides the interface to CAM physics package
-  !
-  ! Revision history:
-  ! Aug  2005,  E. B. Kluzek,  Creation of module from physpkg subroutine
-  ! 2005-10-17  B. Eaton       Add contents of inti.F90 to phys_init().  Add
-  !                            initialization of grid info in phys_state.
-  ! Nov 2010    A. Gettelman   Put micro/macro physics into separate routines
-  ! July 2015   B. Singh       Added code for unified convective transport
-  !-----------------------------------------------------------------------
-
-
-  use shr_kind_mod,     only: i8 => shr_kind_i8, r8 => shr_kind_r8
-  use shr_sys_mod,      only: shr_sys_irtc
-  use shr_sys_mod,      only: shr_sys_flush
-  use spmd_utils,       only: masterproc, iam
-  use physconst,        only: latvap, latice, rh2o
-  use physics_types,    only: physics_state, physics_tend, physics_state_set_grid, &
-       physics_ptend, physics_tend_init,    &
-       physics_type_alloc, physics_ptend_dealloc,&
-       physics_state_alloc, physics_state_dealloc, physics_tend_alloc, physics_tend_dealloc
-  use physics_update_mod,  only: physics_update, physics_update_init, hist_vars, nvars_prtrb_hist, get_var
-  use phys_grid,        only: get_ncols_p, print_cost_p, update_cost_p
-  use phys_gmean,       only: gmean_mass
-  use ppgrid,           only: begchunk, endchunk, pcols, pver, pverp, psubcols
-  use constituents,     only: pcnst, cnst_name, cnst_get_ind!, species_class
-  use camsrfexch,       only: cam_out_t, cam_in_t
-
-  use cam_control_mod,  only: ideal_phys, adiabatic
-  use phys_control,     only: phys_do_flux_avg, phys_getopts, waccmx_is
-  use zm_conv,          only: do_zmconv_dcape_ull => trigdcape_ull
-  use scamMod,          only: single_column, scm_crm_mode
-  use flux_avg,         only: flux_avg_init
+  !-----------------------------------------------------------------------------
+  ! Purpose: Provides the interface to MMF physics package
+  !-----------------------------------------------------------------------------
 #ifdef SPMD
   use mpishorthand
 #endif
-  use perf_mod
-  use cam_logfile,     only: iulog
-  use camsrfexch,      only: cam_export
-
-  use modal_aero_calcsize,    only: modal_aero_calcsize_init, &
-                                    modal_aero_calcsize_reg, &
+    use perf_mod
```

Comment:
you may have added a space here by mistake.

Reply:
fixed.